src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx

```diff
@@ -200,9 +206,8 @@ const BuilderBar = styled.div`
   margin-bottom: 1em;
 `;
 
-const BuilderSelect = styled(SelectInput)`
-  padding: 0.5em;
-  margin-right: 5px;
+const BuilderSelect = styled(SelectField)`
+  margin-right: 10px;
```

Comment:
unless you absolutely _need_ 10px use `space(1)`.

Reply:
This component doesn't use the new spacing units yet, 10px looks a little bit closer to the spacing between the other components than 8 :(